src/components/shot-me/sections/ShotMeNumberEntryScreen.js

```diff
@@ -43,6 +44,16 @@ export default class ShotMeNumberEntryScreen extends React.Component {
   }
 
   render() {
+    if (this.state.loading) {
```

Comment:
please create a new component: LoadingDots, becase we will reuse this component in another screen